src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -819,8 +820,9 @@ services:
 
     form.type.cms_page:
         class: 'PrestaShopBundle\Form\Admin\Improve\Design\Pages\CmsPageType'
+        parent: 'form.type.translatable.aware'
+        public: true
```

Comment:
Seems that having abstract translator requires way more configuration than just injecting it. Is it really worth using abstract translator aware type? :thinking:

Reply:
This is not necessary (I can add just the 2 needed arguments: `@translator` and `locales` but since it was already being used a lot, I did it in order to make the services definition consistent